res/css/structures/_RoomSearch.scss

```diff
@@ -97,12 +97,15 @@ limitations under the License.
     }
 
     &.mx_RoomSearch_minimized {
-        height: auto;
-        width: auto;
-        padding: 5px;
+        height: 32px;
+        min-height: 32px;
```

Comment:
Why do you need the two here? Wouldn't `height` by itself be enough?

Reply:
It won't

![image](https://user-images.githubusercontent.com/2403652/154456075-eb89f4b6-ea15-463c-bba0-07147517d74e.png)

Could flex-basis auto but constraining the height seems easier to reason about